apps/tunes/forms.py

```diff
@@ -0,0 +1,10 @@
+from django import forms
+
+from tunes.models import Emotion
+
+
+class BrowseSongsForm(forms.Form):
+    """Provides validation for /tunes/browse/"""
+
+    emotion = forms.ChoiceField(choices=Emotion.EMOTION_NAME_CHOICES)
+    jitter = forms.FloatField(min_value=0, max_value=1, required=False)
```

Comment:
Let's also add a `limit` parameter to this form so we can take in an arbitrary number of songs to fetch. This should be reasonably constrained, to a max of 50